prisma/migrations/migration_lock.toml

```diff
@@ -0,0 +1,3 @@
+# Please do not edit this file manually
+# It should be added in your version-control system (i.e. Git)
+provider = "postgresql"
```

Comment:
What's this file for?

Reply:
this is used for keeping track of if providers (ex. postgresql) were changed